packages/frontend/src/pages/ViewSqlChart.tsx

```diff
@@ -82,78 +88,91 @@ const ViewSqlChart = () => {
             withFullHeight
             header={<Header mode="view" />}
         >
-            <Stack h="100%" spacing={0}>
-                <Paper
-                    shadow="none"
-                    radius={0}
-                    px="md"
-                    pb={0}
-                    pt="sm"
-                    sx={{
-                        flex: 1,
-                    }}
-                >
-                    <Tabs
+            <Stack h="100%" px="md" py={6}>
+                <Group position="apart">
```

Comment:
could we bring back the `Paper`? I think it looks better if there's a background white added 🤔

Reply:
done ✅